Packages/com.unity.inputsystem/InputSystem/Plugins/UI/UIInputModule.cs

```diff
@@ -87,6 +98,21 @@ internal void ProcessMouse(ref MouseModel mouseState)
             mouseState.OnFrameFinished();
         }
 
+        // if we are using a MultiplayerEventSystem, ignore any transforms
+        // not under the current MultiplayerEventSystem's root.
+        private bool MouseShouldIgnoreTransform(Transform t)
```

Comment:
Thins one should probably be `PointerShouldIgnoreTransform`. Could be pen or touch, too.